### PR TITLE
Changed notebook

### PR DESCRIPTION
**Pull request by avokin2**

### CODE REVIEW

**Comment by avokin2** [comment]
Test

File: notebooks/Boundary Value Problem.ipynb [hunk 1]
```diff
@@ -29,6 +29,20 @@
     "# Код решения (Python3)"
    ]
```
Comment (bozhnyukAlex) [inline notebooks/Boundary Value Problem.ipynb:30]: Comment test

File: notebooks/Boundary Value Problem.ipynb [hunk 2]
```diff
@@ -37,6 +51,7 @@
    "outputs": [],
```
Comment (bozhnyukAlex) [inline notebooks/Boundary Value Problem.ipynb:51]: Comment test 2

File: notebooks/Boundary Value Problem.ipynb [hunk 3]
```diff
@@ -52,7 +67,6 @@
     "    x = [a + k * h for k in range(0, N + 1)]\n",
     "    L = [-1, 0] # we don't use L[0]\n",
     "    K = [-1, y0] # we don't use K[0] \n",
-    "    # L[k] and K[k] evaluation\n",
     "    for j in range(2, N + 1):\n",
     "        ap = 1 - p(x[j - 1]) * h / 2\n",
```
Comment (bozhnyukAlex) [inline notebooks/Boundary Value Problem.ipynb:71]: Comment test 3

File: notebooks/Boundary Value Problem.ipynb [hunk 4]
```diff
@@ -37,6 +51,7 @@
    "outputs": [],
    "source": [
     "from math import sin, cos\n",
+    "#added comment\n",
```
Comment (bozhnyukAlex) [inline notebooks/Boundary Value Problem.ipynb:54]: Test

File: notebooks/Boundary Value Problem.ipynb [hunk 5]
```diff
@@ -37,6 +51,7 @@
    "outputs": [],
    "source": [
     "from math import sin, cos\n",
```
Comment (bozhnyukAlex) [inline notebooks/Boundary Value Problem.ipynb:53]: Test

File: notebooks/Boundary Value Problem.ipynb (hunk 4 above)
Comment (bozhnyukAlex) [inline notebooks/Boundary Value Problem.ipynb:54]: Test1

File: notebooks/Boundary Value Problem.ipynb (hunk 3 above)
Comment (bozhnyukAlex) [inline notebooks/Boundary Value Problem.ipynb:71]: Test

File: notebooks/Boundary Value Problem.ipynb [hunk 6]
```diff
@@ -29,6 +29,20 @@
...
+    "#added cell"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
   {
```
Comment (bozhnyukAlex) [inline notebooks/Boundary Value Problem.ipynb:46]: Left2

File: notebooks/Boundary Value Problem.ipynb [hunk 7]
```diff
@@ -52,7 +67,6 @@
     "    x = [a + k * h for k in range(0, N + 1)]\n",
     "    L = [-1, 0] # we don't use L[0]\n",
     "    K = [-1, y0] # we don't use K[0] \n",
-    "    # L[k] and K[k] evaluation\n",
     "    for j in range(2, N + 1):\n",
     "        ap = 1 - p(x[j - 1]) * h / 2\n",
     "        bp = h * h * q(x[j - 1]) - 2\n",
```
Comment (bozhnyukAlex) [inline notebooks/Boundary Value Problem.ipynb:72]: Another

File: notebooks/Boundary Value Problem.ipynb [hunk 8]
```diff
@@ -52,7 +67,6 @@
     "    x = [a + k * h for k in range(0, N + 1)]\n",
     "    L = [-1, 0] # we don't use L[0]\n",
     "    K = [-1, y0] # we don't use K[0] \n",
```
Comment (bozhnyukAlex) [inline notebooks/Boundary Value Problem.ipynb:69]: Left

Comment (bozhnyukAlex) [inline notebooks/Boundary Value Problem.ipynb:69]: Another

File: notebooks/Boundary Value Problem.ipynb [hunk 9]
```diff
@@ -52,7 +67,6 @@
...
-    "    # L[k] and K[k] evaluation\n",
```
Comment (bozhnyukAlex) [inline notebooks/Boundary Value Problem.ipynb:55]: Example

File: notebooks/Boundary Value Problem.ipynb [hunk 10]
```diff
@@ -52,7 +67,6 @@
...
     "    for j in range(2, N + 1):\n",
```
Comment (bozhnyukAlex) [inline notebooks/Boundary Value Problem.ipynb:70]: Comment

File: notebooks/Boundary Value Problem.ipynb (hunk 3 above)
Comment (bozhnyukAlex) [inline notebooks/Boundary Value Problem.ipynb:71]: Comment

Comment (bozhnyukAlex) [inline notebooks/Boundary Value Problem.ipynb:71]: Comment

File: notebooks/Boundary Value Problem.ipynb (hunk 9 above)
Comment (bozhnyukAlex) [inline notebooks/Boundary Value Problem.ipynb:55]: Check

File: notebooks/Boundary Value Problem.ipynb (hunk 3 above)
Comment (bozhnyukAlex) [inline notebooks/Boundary Value Problem.ipynb:71]: New comment

File: notebooks/Boundary Value Problem.ipynb [hunk 11]
```diff
@@ -52,7 +67,6 @@
     "    x = [a + k * h for k in range(0, N + 1)]\n",
```
Comment (bozhnyukAlex) [inline notebooks/Boundary Value Problem.ipynb:67]: Comment

File: notebooks/Boundary Value Problem.ipynb (hunk 6 above)
Comment (bozhnyukAlex) [inline notebooks/Boundary Value Problem.ipynb:46]: Review old

File: notebooks/Boundary Value Problem.ipynb [hunk 12]
```diff
@@ -29,6 +29,20 @@
     "# Код решения (Python3)"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
```
Comment (bozhnyukAlex) [inline notebooks/Boundary Value Problem.ipynb:36]: Test

File: notebooks/Boundary Value Problem.ipynb (hunk 5 above)
Comment (bozhnyukAlex) [inline notebooks/Boundary Value Problem.ipynb:53]: Test1

File: notebooks/Boundary Value Problem.ipynb (hunk 9 above)
Comment (bozhnyukAlex) [inline notebooks/Boundary Value Problem.ipynb:55]: Test

Comment (bozhnyukAlex) [inline notebooks/Boundary Value Problem.ipynb:55]: Test1

File: notebooks/Boundary Value Problem.ipynb [hunk 13]
```diff
@@ -52,7 +67,6 @@
     "    x = [a + k * h for k in range(0, N + 1)]\n",
     "    L = [-1, 0] # we don't use L[0]\n",
```
Comment (bozhnyukAlex) [inline notebooks/Boundary Value Problem.ipynb:68]: Review

File: notebooks/Boundary Value Problem.ipynb (hunk 3 above)
Comment (bozhnyukAlex) [inline notebooks/Boundary Value Problem.ipynb:71]: Another review

File: notebooks/Boundary Value Problem.ipynb (hunk 8 above)
Comment (bozhnyukAlex) [inline notebooks/Boundary Value Problem.ipynb:69]: Comment

File: notebooks/Boundary Value Problem.ipynb (hunk 4 above)
Comment (bozhnyukAlex) [inline notebooks/Boundary Value Problem.ipynb:54]: Comment2

Comment (bozhnyukAlex) [inline notebooks/Boundary Value Problem.ipynb:54]: Comment2

Comment (bozhnyukAlex) [inline notebooks/Boundary Value Problem.ipynb:54]: C

Comment (bozhnyukAlex) [inline notebooks/Boundary Value Problem.ipynb:54]: Com

File: notebooks/Boundary Value Problem.ipynb (hunk 13 above)
Comment (bozhnyukAlex) [inline notebooks/Boundary Value Problem.ipynb:68]: Comment
